.github/workflows/main.yml

```diff
@@ -1,6 +1,8 @@
-name: Mask CI
+name: CI
 
 on:
+  push:
```

Comment:
Problems like this:
```
Error: projects/ngx-mask-lib/src/test/basic-logic.spec.ts:352:5 - error TS2322: Type 'null' is not assignable to type 'string'.

352     component.mask = null;
        ~~~~~~~~~~~~~~
```
could be detected much easier if we turn on the CI on push (meaning if something is pushed without PR)